spec/shipcloud/carrier_spec.rb

```diff
@@ -24,7 +24,7 @@
   describe '.all' do
     it 'makes a new Get request using the correct API endpoint' do
       expect(Shipcloud).to receive(:request).
-        with(:get, 'carriers', {}).and_return([])
+        with(:get, 'carriers', {}, api_key: nil).and_return([])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.